src/odemis/gui/util/img.py

```diff
@@ -2658,3 +2658,37 @@ def merge_screen(ima, imb):
     out.metadata = md
     return out
 
+
+def assemble_z_cube(images, zlevels):
```

Comment:
Please move it to `odemis.util.img`, not in the GUI. One of the goal is that the "acq" module doesn't depend on the GUI. So the GUI util only contains code that is related to the GUI itself (like wxpython). 
I think the name should then be `assembleZCube`, to follow the convention in that module.